src/icons/annotations/IconRegionAnnotation.js.flow

```diff
@@ -0,0 +1,25 @@
+// @flow
+import * as React from 'react';
+import { white } from '../../styles/variables';
+
+import AccessibleSVG from '../accessible-svg';
+
+import type { Icon } from '../flowTypes';
+
+const IconRegionAnnotation = ({ className = '', color = white, height = 21, title, width = 22 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-annotation-region ${className}`}
```

Comment:
Not sure if it matters for `.flow` files, but `bdl-IconRegionAnnotation` for consistency?